lookout/style/format/features.py

```diff
@@ -151,6 +148,31 @@ def __init__(self, language: str, siblings_window: int = 5, parents_depth: int =
         self.tokens = importlib.import_module("lookout.style.format.langs.%s.tokens" % language)
         self.roles = importlib.import_module("lookout.style.format.langs.%s.roles" % language)
 
+        # Order is important and should be consistent with _inplace_write_vnode_features function
+        # where features generation happens.
+        self.feature_layout = OrderedDict([
+            ("self", FeaturesGroup(("start_line", "start_col"), None)),
+            ("left_siblings", FeaturesGroup(("start_line_diff", "end_line_diff", "start_col_diff",
+                                             "end_col_diff", "role_id"), siblings_window)),
+            ("right_siblings", FeaturesGroup(("role_id", ), siblings_window)),
+            ("parents", FeaturesGroup(("role_id", ), parents_depth)),
+        ])
+        self.feature_names = self._init_feature_names(self.feature_layout)
```

Comment:
Make it a normal method and don't pass in `self.feature_layout`

Reply:
So you can set feature_names and feature2index inside

Reply:
Mmm, ok, I fit it, but can you explain why? I was thinking about both options when I write a code and decide that this one is more transparent for a reader.